plugin.json

```diff
@@ -21,8 +21,110 @@
+                        "value": "uuid"
+                    },
+                    {
+                        "display_name": "Mattermost context specific names. Combination of team name, channel name, and random text in Public and Private channels; personal meeting name in Direct and Group Message channels.",
+                        "value": "mattermost"
+                    },
+                    {
+                        "display_name": "Allow user to select meeting name",
+                        "value": "ask"
+                    }
+                ]
+            },
+            {
+                "key": "JitsiJWT",
+                "display_name": "Use JWT Authentication for Jitsi:",
+                "type": "bool",
+                "help_text": "(Optional) If your Jitsi server uses JSON Web Tokens (JWT) for authentication, set this value to true."
+            },
+            {
+                "key": "JitsiAppID",
+                "display_name": "App ID for JWT Authentication:",
+                "type": "text",
+                "help_text": "(Optional) The app ID used for authentication by the Jitsi server and JWT token generator."
+            },
+            {
+                "key": "JitsiAppSecret",
+                "display_name": "App Secret for JWT Authentication:",
+                "type": "text",
+                "help_text": "(Optional) The app secret used for authentication by the Jitsi server and JWT token generator."
+            },
+            {
+                "key": "JitsiLinkValidTime",
+                "display_name": "Meeting Link Expiry Time (minutes):",
+                "type": "number",
+                "help_text": "(Optional) The number of minutes from when the meeting link is created to when it becomes invalid. Minimum is 1 minute. Only applies if using JWT authentication for your Jitsi server.",
+                "default": 30
+            },
+            {
+                "key": "JaaSAppID",
+                "display_name": "AppID for JaaS Authentication:",
+                "type": "text",
+                "help_text": "Specify your JaaS AppID. You can get the AppID from https://jaas.8x8.vc/#/apikeys."
+            },
+            {
+                "key": "JaaSApiKey",
+                "display_name": "Api key ID for JaaS Authentication:",
+                "type": "text",
+                "help_text": "Specify your JaaS Api key ID. You can get the Api key ID from https://jaas.8x8.vc/#/apikeys."
+            },
+            {
+                "key": "JaaSPrivateKey",
+                "display_name": "RSA Private key for JaaS Authentication:",
+                "type": "longtext",
+                "help_text": "Specify your JaaS private key. You can get the private key by generating an API key pair from https://jaas.8x8.vc/#/apikeys OR from https://jaas.8x8.vc/#/start-guide in the Api Key section. (NOTE: While generating the key manually make sure to set the size to be 2048 and not 4096)."
             }
```

Comment:
Since there's no longer a custom setting, I think the `components/admin_settings` folder can be removed